.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,61 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+name: "CodeQL"
+
+on:
+  push:
+    branches: [0.27-maintenance, main]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [0.27-maintenance, main]
+  schedule:
+    - cron: '0 8 * * 4'
```

Comment:
@kevinbackhouse sorry I missed this part on my first review, but why would we need this scheduled run on top of push & pull request?

Reply:
@hassec: Good point. That was cut 'n paste from me, and you're right that we don't need it.